docs/pages/components/Popover.mdx

```diff
@@ -5,7 +5,24 @@ source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/co
 ---
 
 import { useRef, useState } from 'react'
-import { Popover, Button } from '@deque/cauldron-react'
+import { Select, Popover, Button } from '@deque/cauldron-react'
+export const placements = [
```

Comment:
Nit: Since these placements are inherited from [Anchored overlay](https://github.com/dequelabs/cauldron/blob/develop/docs/pages/components/AnchoredOverlay.mdx), we might be able to export them from its doc page and pull them in here?

```tsx
import { placements } from './AnchoredOverlay.mdx'
```

Not a huge deal if this doesn't work, but just a suggestion if it does.